.storybook/preview-head.html

```diff
@@ -1,2 +1,5 @@
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.1.1/themes/algolia.min.css">
-<link rel="stylesheet" href="storybook.css">
+<link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.3.0/themes/algolia.min.css"
```

Comment:
InstantSearch.css 7.3.0 is damaged and doesn't include the changes made in 7.2.0 (previous button styles).

- 7.2.0: https://unpkg.com/instantsearch.css@7.2.0/themes/algolia.css
- 7.3.0: https://unpkg.com/instantsearch.css@7.3.0/themes/algolia.css

We released a new patch including the fix.

```suggestion
  href="https://cdn.jsdelivr.net/npm/instantsearch.css@7.3.1/themes/algolia.min.css"
```